extensions/vscode/package.json

```diff
@@ -0,0 +1,134 @@
+  "publisher": "StackBlitz",
+  "version": "0.0.11",
+  "engines": {
+    "vscode": "^1.80.0"
+  },
+  "repository": {
+    "url": "https://github.com/stackblitz/tutorialkit"
+  },
+  "categories": [
+    "Other"
+  ],
+  "activationEvents": [
+    "onStartupFinished"
+  ],
+  "main": "./dist/extension.js",
+  "contributes": {
+    "commands": [
+      {
+        "command": "tutorialkit.select-tutorial",
+        "title": "Select Tutorial",
+        "icon": "$(book)"
+      },
+      {
+        "command": "tutorialkit.add-lesson",
+        "title": "Add Lesson"
+      },
+      {
+        "command": "tutorialkit.add-chapter",
+        "title": "Add Chapter"
+      },
+      {
+        "command": "tutorialkit.add-part",
+        "title": "Add Part"
+      },
+      {
+        "command": "tutorialkit.refresh",
+        "title": "Refresh Lessons",
+        "icon": "$(refresh)"
+      }
+    ],
+    "viewsWelcome": [
+      {
+        "view": "tutorialkit-lessons-tree",
+        "id": "tutorialkit-splashscreen",
+        "contents": "",
+        "title": "Tutorial",
+        "description": "TutorialKit",
+        "when": "!tutorialkit:initialized"
+      },
+      {
+        "view": "tutorialkit-lessons-tree",
+        "id": "tutorialkit-splashscreen",
+        "contents": "Looks like there is no TutorialKit project in this workspace.\n[Rescan workspace](command:tutorialkit.initialize?true)",
+        "title": "Tutorial",
+        "description": "TutorialKit",
+        "when": "tutorialkit:initialized && !tutorialkit:multiple-tutorials"
+      },
+      {
+        "view": "tutorialkit-lessons-tree",
+        "id": "tutorialkit-multiple-tutorials",
+        "contents": "Welcome to TutorialKit!\nLooks like there is more than one tutorial in your workspace.\n[Select a tutorial](command:tutorialkit.select-tutorial)",
+        "title": "Tutorial",
+        "description": "TutorialKit",
+        "when": "tutorialkit:multiple-tutorials"
+      }
+    ],
+    "views": {
+      "explorer": [
+        {
+          "id": "tutorialkit-lessons-tree",
+          "name": "Tutorial",
+          "visibility": "visible",
+          "initialSize": 3
+        }
+      ]
+    },
+    "menus": {
+      "view/title": [
+        {
+          "command": "tutorialkit.select-tutorial",
+          "when": "view == tutorialkit-lessons-tree && tutorialkit:multiple-tutorials",
+          "group": "navigation"
+        },
+        {
+          "command": "tutorialkit.refresh",
+          "when": "view == tutorialkit-lessons-tree && tutorialkit:tree",
+          "group": "navigation"
+        }
+      ],
+      "view/item/context": [
+        {
+          "command": "tutorialkit.add-lesson",
+          "when": "view == tutorialkit-lessons-tree && viewItem == chapter"
+        },
+        {
+          "command": "tutorialkit.add-chapter",
+          "when": "view == tutorialkit-lessons-tree && viewItem == part"
+        }
+      ]
+    }
+  },
+  "scripts": {
```

Comment:
Shall we move the scripts a little higher so it's faster to find these? Wondering because I'd imagine that scripts are more frequently used than the other props above it. WDYT?

Reply:
Not when I was developing it. I'd reference the `contribute` all the time and would barely look at `scripts`.

Reply:
Interesting. Personally I have the scripts always at the top but prolly subjective.

Reply:
I agree with @sulco, I'll keep it with `contributes` first :+1: